src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocks.java

```diff
@@ -31,6 +36,45 @@ public int doSideEffect() {
   }
 
   public void emptyMethod() {}
+
+  void foo() throws Exception {
+    int a = 90;
+
+    if (a == 1) {
+    } else {} // false-negative
```

Comment:
Please create separate issue on this and catch.
We will think what we can do to address such false negatives, but will be low priority.
Just reference new issue in table to let users be aware.

Reply:
issue opened: https://github.com/checkstyle/checkstyle/issues/15664, referenced new issue in table: 

https://github.com/checkstyle/checkstyle/blob/4047413e6d95f355c559f9a7ad09d8df2a59656f/src/xdocs/google_style.xml#L706-L708

Reply:
`// false-negative until #15664 `

Reply:
Please do same for all other case like this.
To make it clear we saw problem, and there is issue on this